packages/viewer/src/scenes/camera.ts

```diff
@@ -50,6 +110,15 @@ export class Camera implements FrameCamera.FrameCamera {
     return this.update({ lookAt, position });
   }
 
+  /**
+   *
```

Comment:
Might be worth adding a description/example to this method similar to what we have for the `scene.items` method